pep-0001.txt

```diff
@@ -659,6 +660,9 @@ archives, rather than just a mailto: or hyperlink to the list itself.
 The Type header specifies the type of PEP: Standards Track,
 Informational, or Process.
 
+The optional Topic header specifies which topic, if any, the PEP belongs in.
+See https://peps.python.org/topic/ for the existing topics.
```

Comment:
```suggestion
See the :ref:`topic-index` for the existing topics.
```

Use a proper (and named) ref target here, rather than a hardcoded external URL (you'll need to add a ref target label to the topic index page, i.e.:

```rst
.. _topic-index:

Topic Index
***********
```

(You could also use `:doc:` without that, but `:ref:` is preferred as it works regardless of the location of the source or destination file, and won't break if we ever move either).

Reply:
Thanks, done with `:ref:`